docs/tutorials/connecting-EL-with-T-using-SQL.md

```diff
@@ -0,0 +1,288 @@
+off in the destination settings page. 
+
+This could be useful if:
+1. you have different usage than analytics that could be handled with these initial data in raw JSON format.
+1. you can implement your own Transformer (even in a different language such as Java or in Spark for example, or another transformation tool: DBT or Dataform)
+1. you want to customize and change how the data is normalized with your own queries (add deduplication logic since Airbyte is not doing it natively yet?)
+
+In order to do so, we will now describe how you can leverage the basic normalization outputs that Airbyte 
+generates to build your own transformations if you don't want to start from scratch.
+
+Note: We will rely on docker commands that we've gone over as part of another [Tutorial on Exploring Docker Volumes](exploring-workspace-folder.md).
+
+## (Optional) Configure some Covid \(data\) source and Postgres destinations
+
+If you have sources and destinations already setup on your instance, you can skip to the next section.
+
+For the sake of this tutorial, let's create some source and destination as an example that we can refer to afterward.
+We'll be using a file accessible from a public API, so you can easily reproduce this setup:
+
+```text
+Here are some examples of public API CSV:
+https://storage.googleapis.com/covid19-open-data/v2/latest/epidemiology.csv
+```
+
+![](../.gitbook/assets/connecting-EL-with-T-1.png)
+
+And a local Postgres Database, making sure that "Basic normalization" is enabled:
+
+![](../.gitbook/assets/connecting-EL-with-T-2.png)
+
+After setting up the connectors, we can trigger the sync and study the logs:
+
+![](../.gitbook/assets/connecting-EL-with-T-3.png)
+
+Notice that the process ran in the `/tmp/workspace/5/0` folder.
+
+## Identify Workspace ID with Normalize steps
+
+If you went through the previous setup of source/destination section and run a sync, you were able 
+to identify which workspace was used, let's define some environment variables to remember this:
+
+```bash
+NORMALIZE_WORKSPACE="5/0/"
+```
+
+Or if you want to find any folder where the normalize step was run:
+
+```bash
+# find automatically latest workspace where normalization was run
+NORMALIZE_WORKSPACE=`docker run --rm -i -v airbyte_workspace:/data  busybox find /data -path "*normalize/models*" | sed -E "s;/data/([0-9]+/[0-9]+/)normalize/.*;\1;g" | sort | uniq | tail -n 1`
+```
+
+## Export Plain SQL files
+
+Airbyte is using internally a tool for handling transformations. 
+
+It is a python-based program that reads the `catalog.json` file and generates code on how to interpret and transform it.
+
+The final output of such a tool is producing SQL files that can be run on top of the destination that you selected.
+
+Therefore, it is possible to extract these SQL files, modify them and run it yourself manually outside of Airbyte!
+
+You would be able to find these at the following location inside the docker containers: 
```

Comment:
which docker container? i know it's technically accessible from several, but probably best to just pick one so the user has something concrete to follow.